src/wp-includes/post.php

```diff
@@ -417,6 +418,7 @@ function create_initial_post_types() {
 			'description'           => __( 'Template parts to include in your templates.' ),
 			'public'                => false,
 			'_builtin'              => true, /* internal use only. don't use this when registering your own post type. */
+			'_edit_link'            => 'site-editor.php?postType=%s&postId=%s&canvas=edit', /* internal use only. don't use this when registering your own post type. */
```

Comment:
Please see the comment above.
```suggestion
			'_edit_link'            => $edit_link, /* internal use only. don't use this when registering your own post type. */
```